index.html

```diff
@@ -1,11 +1,42 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>okej</title>
-</head>
-<body>
-    
-</body>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+<div>
+<a href="">mydło |</a>
+<a href="">lodówka |</a>
+<a href="">krzesło |</a>
+</div>
+
+<div>
+    <p>
+    <a href="">kosiarka</a> <br><br>

```

Comment:
Inna mozliwosc:

```suggestion
    <p><a href="">kosiarka</a></p>
```

To bedzie dzialac, chociaz znacznik `<p>` sluzy do innego celu. Jest to akapit i najlepiej jak bedzie zawieral tekst. Dlatego tez mozna to tez zrobic w ten sposob:

```suggestion
    <div><a href="">kosiarka</a></div>
    <div><a href="">pralka</a></div>
```

Reply:
tutaj poprawiłem tylko nie wiem jak w tym ostatnim sprawić żeby te słowa występowały jako linki